frontend/src/APIClients/LogRecordAPIClient.ts

```diff
@@ -119,8 +118,60 @@ const createLog = async (
   }
 };
 
+const deleteLogRecord = async (logId: number): Promise<boolean> => {
+  try {
+    const bearerToken = `Bearer ${getLocalStorageObjProperty(
+      AUTHENTICATED_USER_KEY,
+      "accessToken",
+    )}`;
+    await baseAPIClient.delete(`/log_records/${logId}`, {
+      headers: { Authorization: bearerToken },
+    });
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+const editLogRecord = async (
+  logId: number,
+  userId: number,
+  residentId: number,
+  flagged: boolean,
+  note: string,
+  tags: string[],
+  building: string,
+  attentionTo?: number,
+): Promise<any> => {
```

Comment:
Can we update the return type (I think it's just `LogRecord` but could be wrong)

Reply:
Discussed with @owen-sellner  and I think we should just simply update the type to be a `boolean` to make it consistent with `deleteLogRecord` ; both of these services simply return a 201 with a message attached

![Screen Shot 2023-07-24 at 8 44 53 PM](https://github.com/uwblueprint/supportive-housing/assets/53386475/928fea9a-5b3d-40bb-8cc2-3acb8e58fd99)

However, the message isn't very helpful, and we already implemented a `boolean` return type for `deleteLogRecord` ; I think keeping it consistent is fine here